msg_publisher.py

```diff
@@ -0,0 +1,24 @@
+import zmq
+import logging
+
+from config import Config
+
+
+class MsgPublisher(object):
+    def __init__(self, zmq_socket):
+        self.zmq_socket = zmq_socket
+        self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.DEBUG)
+
+    def publish(self, message):
+        self.zmq_socket.send_json(message)
```

Comment:
Okay, I might know why you dis not using the tornado ioloop. Is it the callback?